test/suites/multi_arch.bash

```diff
@@ -36,17 +36,32 @@ SUITE_multi_arch() {
     expect_stat direct_cache_hit 2
     expect_stat cache_miss 3
 
-    # A single -Xarch_* matching -arch is supported.
+    # A multiple -Xarch_* matching -arch are supported.
```

Comment:
While that's true, perhaps the comment no longer makes sense in this context since it only tests one `-Xarch` option? Maybe remove the comment instead?

Reply:
Done.